subprojects/jacoco/src/integTest/groovy/org/gradle/testing/jacoco/plugins/JacocoVersionIntegTest.groovy

```diff
@@ -43,7 +43,7 @@ class JacocoVersionIntegTest extends MultiVersionIntegrationSpec {
         then:
         def report = htmlReport()
         report.totalCoverage() == 100
-        report.jacocoVersion() == version
+        report.jacocoVersion() == version || report.jacocoVersion().startsWith(version)
```

Comment:
The latest JaCoCo library version has a non-timestamped version but they still use the timestamp when generating the report. Doh! I think it's simply an oversight on their end.